pkg/cloudprovider/types.go

```diff
@@ -83,6 +88,20 @@ type InstanceType struct {
 
 type InstanceTypes []*InstanceType
 
+func (its InstanceTypes) FilterByMinValues(reqs scheduling.Requirements) (InstanceTypes, error) {
```

Comment:
thoughts on making cloudprovider/instancetypes.go to separate out some of this logic?

Reply:
Eh, Im personally a fan of having the types.go file have the cloudprovider interface + instance types + respective member functions + enums